packages/core/test/plugin-progress-bar.config.mock.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * This config file is here to demonstrate the progress bar for plugins
+ *
+ * Usage:
+ * npx ./dist/packages/cli collect --config=./packages/core/test/plugin-progress-bar.config.mock.ts
+ */
+import { join } from 'path';
+
+// Small hack to control the number of plugins while debugging
+const numPlugins = parseInt(
+  process.argv
+    .find(arg => arg.includes('numPlugins='))
+    ?.split('=')
+    .pop() || '6',
+);
+
+const outputDir = './tmp';
+const pluginProcess = join(
+  'packages',
+  'core',
+  'test',
+  `plugin-progress-bar-plugin-process.mock.mjs`,
+);
```

Comment:
This way it'll be independent of CWD:

```suggestion
const pluginProcess = join(
  fileURLToPath(dirname(import.meta.url)),
  'plugin-progress-bar-plugin-process.mock.mjs',
);
```